Makefile

```diff
@@ -103,14 +104,13 @@ mod-download:
 ## --------------------------------------
 
 .PHONY: gen
-gen: mod-download proto/agent/agent_grpc.pb.go proto/agent/agent.pb.go konnectivity-client/proto/client/client_grpc.pb.go konnectivity-client/proto/client/client.pb.go mock_gen
+gen: mod-download gen-proto mock_gen
 
-konnectivity-client/proto/client/client_grpc.pb.go konnectivity-client/proto/client/client.pb.go: konnectivity-client/proto/client/client.proto
+.PHONY: gen-proto
+gen-proto:
```

Comment:
bin/proxy-agent and bin/proxy-server seem like they have a similar premature optimization issue.

Reply:
Agreed, https://github.com/kubernetes-sigs/apiserver-network-proxy/pull/482 already fixed that one.